Configure

```diff
@@ -1323,8 +1323,8 @@ if (defined($config{api})) {
 if ($strict_warnings)
 	{
 	my $wopt;
-	die "ERROR --strict-warnings requires gcc or clang"
-            unless $ecc eq 'gcc' || $ecc eq 'clang';
+	die "ERROR --strict-warnings requires gcc , clang or icc"
```

Comment:
No space before comma, and I'd suggest the Oxford comma as well:

`die "ERROR --strict-warnings requires gcc, clang, or icc"`